### PR TITLE
Add build instructions for deploy

### PR DESCRIPTION
**Pull request by gerth2**
Because I always forget the command

### CODE REVIEW

File: source/docs/contributing/photonvision/build-instructions.rst [hunk 1]
```diff
@@ -93,6 +93,27 @@ Running the following command under the root directory will build the jar under
...
+
+As a convinenece, the build has built in `deploy` command which builds, deploys, and starts the current source code on a coprocessor. It is generally intended for Raspberry Pi based coprocessors.
+
+An architecture override is required to specify the deploy target's architecture.
+
+.. tab-set::
+
+   .. tab-item:: Linux
+
+      ``./gradlew deploy -PArchOverride=linuxarm64`
```
Comment (mcm001) [inline source/docs/contributing/photonvision/build-instructions.rst:107]: Gradlew clean should be run as a paranoia step before this to make sure that the correct native libraries are pulled in

**Comment by mdurrani808** [comment]
does this only work for raspis or if i put linuxaarch64 (or whatever the incanation) will it work

**Comment by gerth2** [comment]
Only ever been tested on a raspi.... theoretically the clones could work too....

**Comment by mcm001** [comment]
Should work on any aarch64 target.

**Comment by gerth2** [comment]
Still relies on linux commands and systemctl.... I'd tentatively say "tested on a pi" - if someone had a windows coproc I'd say it won't work for sure, need to review.

**Review by mdurrani808 — APPROVED** [review]
lgtm ship it

**Pull request merged**